saleor/plugins/webhook/tests/test_webhook.py

```diff
@@ -209,6 +210,42 @@ def test_checkout_updated(mocked_webhook_trigger, settings, checkout_with_items)
     )
 
 
+@mock.patch("saleor.plugins.webhook.plugin.trigger_webhooks_for_event.delay")
+def test_page_created(mocked_webhook_trigger, settings, page):
+    settings.PLUGINS = ["saleor.plugins.webhook.plugin.WebhookPlugin"]
+    manager = get_plugins_manager()
+    manager.page_created(page)
+
+    expected_data = generate_page_payload(page)
+    mocked_webhook_trigger.assert_called_once_with(
+        WebhookEventType.PAGE_CREATED, expected_data
+    )
+
+
+@mock.patch("saleor.plugins.webhook.plugin.trigger_webhooks_for_event.delay")
+def test_page_updated(mocked_webhook_trigger, settings, page):
+    settings.PLUGINS = ["saleor.plugins.webhook.plugin.WebhookPlugin"]
+    manager = get_plugins_manager()
+    manager.page_updated(page)
+
+    expected_data = generate_page_payload(page)
+    mocked_webhook_trigger.assert_called_once_with(
+        WebhookEventType.PAGE_UPDATED, expected_data
+    )
+
+
+@mock.patch("saleor.plugins.webhook.plugin.trigger_webhooks_for_event.delay")
+def test_page_deleted(mocked_webhook_trigger, settings, page):
+    settings.PLUGINS = ["saleor.plugins.webhook.plugin.WebhookPlugin"]
+    manager = get_plugins_manager()
+    manager.page_deleted(page)
```

Comment:
To proper test it I would simulate the situation that page is deleted. You can do the similar steps like `ModelDeleteMutation`
```
db_id = instance.id
instance.delete()

instance.id = db_id
```